### PR TITLE
CSS  - Add video responsive class and Geotrek video on Homepage

### PR DESCRIPTION
**Pull request by camillemonchicourt**
- I added a ``video-responsive`` CSS class. Not sure I put it in the right file ?
- I applied it to existing videos in articles.
- I added the Geotrek presentation video on Homepage (with EN subtitles by default on EN and ES versions as we only have EN subtitles available actually) and applied the CSS class to this video.

### CODE REVIEW

File: assets/css/components/_layout.scss [hunk 1]
```diff
@@ -40,6 +40,21 @@ a {
...
 }
 
+.video-responsive { 
+    overflow:hidden; 
+    padding-bottom:56.25%; 
+    position:relative; 
+    height:0;
+}
+
+.video-responsive iframe {
```
Comment (numahell) [inline assets/css/components/_layout.scss]: Sass is used, this could be simplified by using nested properties

**Review by numahell — CHANGES_REQUESTED** [review]
> Not sure I put it in the right file ?

It is the correct file : this CSS class can be used everywhere in the website, not only in news

File: assets/css/components/_layout.scss (hunk 1 above)
Comment (camillemonchicourt) [reply assets/css/components/_layout.scss]: Hum, don't know what nested properties are. But I will try to find.

Comment (camillemonchicourt) [reply assets/css/components/_layout.scss]: OK @numahell, I think it did it with the nested way now.

Comment (numahell) [reply assets/css/components/_layout.scss]: Yeah it does ! sorry I should have give you doc page from sass.

**Review by numahell — APPROVED** [review]

**Comment by camillemonchicourt** [comment]
OK thanks, it is not working properly on mobile : https://geotrek.fr while it was OK in my tests... 🤔
Same here : https://geotrek.fr/2021/11/04/geotrek-rencontres-2021-synthese.html

**Comment by camillemonchicourt** [comment]
It is working properly in the news, was just my cache...
Something seems to be wrong with the video on homepage, I will check.

**Pull request merged**